lambdas/service/app.py

```diff
@@ -106,11 +119,43 @@ def jwt_auth(auth_request) -> AuthResponse:
     return AuthResponse(routes=['*'], principal_id='user', context=context)
 
 
-@app.route('/', cors=True)
-def hello():
-    return {'Hello': 'World!'}
+pkg_root = os.path.dirname(os.path.abspath(__file__))
 
 
+@app.route('/', cors=True)
+def swagger_ui():
+    local_path = os.path.join(pkg_root, 'static')
+    dir_name = local_path if os.path.exists(local_path) else pkg_root
+    with open(os.path.join(dir_name, 'openapi.html')) as f:
+        openapi_ui_html = f.read()
+    return Response(status_code=200,
+                    headers={"Content-Type": "text/html"},
+                    body=openapi_ui_html)
+
+
+@app.route('/openapi', methods=['GET'], cors=True)
+def openapi():
+    gateway_id = os.environ['AZUL_SERVICE_API_GATEWAY_ID']
+    specs = annotated_specs(gateway_id, app, api_info)
+    return Response(status_code=200,
+                    headers={"content-type": "application/json"},
+                    body=specs)
+
+
+@spec(
+    {
+        'parameters': [
+            {
+                'in': 'query',
+                'name': 'foo',
+                'description': 'This is not a real parameter',
+                'required': False,
+                'schema': {'$ref': "#/components/schemas/Empty"}
+            }
+        ],
+        'tags': ['Health']
+    }
+)
 @app.route('/health/basic', methods=['GET'], cors=True)
```

Comment:
And lets consider this 

```suggestion
@app.route('/health/basic', methods=['GET'], cors=True, openapi={
})
```

Reply:
I dig it. The only con I can think of is that it takes some of the OpenAPI logic out of `openapi.py`.

Reply:
I would still prefer the parameter name `spec` over `openapi` however.